set-recording-comments.user.js

```diff
@@ -7,7 +7,9 @@
 // @namespace      790382e7-8714-47a7-bfbd-528d0caa2333
 // @downloadURL    https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/set-recording-comments.user.js
 // @updateURL      https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/set-recording-comments.user.js
-// @include        /^https?:\/\/(\w+\.)?musicbrainz\.org\/release\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}(\/disc\/\d+|$)/
+// @match          *://*.musicbrainz.org/release/*
```

Comment:
Unfortunately `@match` is not flexible enough to handle MB's URLs. The `release/*` wildcard matches way too many path segments which are not MBIDs and which would have to be excluded again. You've got 2 to 4 covered, but there are more, listing them all is not paracticable.
It would be simpler to keep the existing regex and extend it to allow any query parameters with `(\?.+?)?`.

Reply:
Alright, fixed now.